scripts/cluster_utils.py

```diff
@@ -84,6 +85,7 @@ def generate_run_configs(config_filename: str,
         num_seeds = config["NUM_SEEDS"]
         args = config["ARGS"]
         flags = config["FLAGS"]
+        use_gpu = config["USE_GPU"]
```

Comment:
looking great overall! one more medium thing: can we make USE_GPU optional and default to False so that we don't have to modify these existing files? I think we can accomplish this by just checking here if `USE_GPU` is in `config`, and if not, set `use_gpu` to False